src/Hl7.Fhir.Core/Model/FhirPatchParameters.cs

```diff
@@ -0,0 +1,61 @@
+﻿/* 
```

Comment:
I don't see the added value of this class. I rather don't want to add it to the core library.

Reply:
My intent with adding this class was to support `PatchDocument` as a `[FromBody]` input parameter in an ASP.NET Controller method. But current parser implementation only deserializes types derived from `Base`.

```csharp
[HttpPatch]
[Route(KnownRoutes.ResourceTypeById)]
public async Task<IActionResult> Patch(string typeParameter, string idParameter, [FromBody]FhirPatchParameters resource)
```

The benefit from having it as input parameter, rather than constructing it in the method body is implicit validation performed on input paremeters. 

I guess it is ok not addding it to the core library, since the class can be implemented by consumers of the library. Though I am not sure how to do that, since implementing this class required change to `ModelInfo` template ([see](https://github.com/FirelyTeam/fhir-net-api/pull/1375#discussion_r450845167))